test/ui/load40-cache-on.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Loads users/ page 40 times (with caching on) capturing a screenshot and load time for each.
+ */
+
+ const puppeteer = require("puppeteer");
+ const testCount = 40;
+ const url = "https://nsc-thursday-react-app.azureedge.net/users";
+
+ (async () => {
+  const browser = await puppeteer.launch();
+
+  const page = await browser.newPage();
+  page.setCacheEnabled([true]);
```

Comment:
My understanding is that we're *not* using browser caching, we're using the Redis caching - so I *think* you should have this set to False and then test it while CACHE_TOGGLE is set to "True" in the Azure function app. But I'm not sure - I'll approve if you get the screenshots to wait until after the page fully loads. : )